lib/utils/isPathNotFoundError.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const util = require('util');
+
+/**
+ * @param {unknown} error
+ * @returns {error is NodeJS.ErrnoException}
+ */
+module.exports = function isPathNotFoundError(error) {
+	// @ts-expect-error -- TS2339: Property 'code' does not exist on type 'Error'.
+	return util.types.isNativeError(error) && error.code === 'ENOENT';
```

Comment:
[note] It seems there are no functions to check if an error is `NodeJS.ErrnoException`... 🤔 

- https://nodejs.org/api/errors.html#errors_error_code
- https://nodejs.org/api/util.html#util_util_types_isnativeerror_value